File: SimpleMapJava/app/src/main/java/com/woosmap/simplemap/MainActivity.java
package com.woosmap.simplemap;

import androidx.appcompat.app.AppCompatActivity;
import com.woosmap.sdk.MapView;
import com.woosmap.sdk.geometry.LatLng;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    private MapView MapView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        MapView = findViewById(R.id.mapView);
        MapView.setZoomLevel(3);
        MapView.setCenter(new LatLng(47.7724413, 12.4833043));
    }
}

File: SimpleMapKotlin/app/src/main/java/com/woosmap/simplemap/MainActivity.kt
package com.woosmap.simplemap

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.woosmap.sdk.MapView
import com.woosmap.sdk.geometry.LatLng

class MainActivity : AppCompatActivity() {
    private lateinit var mapView: MapView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mapView = findViewById(R.id.mapView)
        mapView.zoomLevel = 3.0
        mapView.center = LatLng(47.7724413, 12.4833043)
    }
}